client/reader/sidebar/index.jsx

```diff
@@ -269,12 +273,31 @@ export class ReaderSidebar extends Component {
 		);
 	}
 
-	render() {
+	renderGlobalSidebar() {
+		const asyncProps = {
+			placeholder: null,
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg)

I guess this prop originates from here: https://github.com/Automattic/wp-calypso/blob/65aa1113c9313690226205f659a9db17268b98b6/client/my-sites/navigation/index.jsx#L22-L26 for `AyncLoad` component. 
Given that we are passing props to the `GlobalSidebar`, it appears that this specific prop `placeholder` may not be necessary. 🙂